client/state/themes/actions/activate.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import { isJetpackSite } from 'state/sites/selectors';
+import { activateTheme } from 'state/themes/actions/activate-theme';
```

Comment:
Any reason why we import this from the direct import vs the re-exported action creator in `state/themes/actions`? Avoiding circular dependencies?

Reply:
Correct. I prefer to import directly from the sibling module than the index to simplify the dependency graph. I don't trust `webpack` not to make a mess of things 😄